### ISSUE TITLE
Error when running in Station mode

### ISSUE THREAD

**Issue by Tremito96** [StevenMHernandez/ESP32-CSI-Tool#10]
![Screenshot 2021-03-26 081548](https://user-images.githubusercontent.com/79624622/112570324-d688b500-8e0b-11eb-9eee-c2e89495876a.png)
![Screenshot 2021-03-26 081618](https://user-images.githubusercontent.com/79624622/112570328-d7b9e200-8e0b-11eb-9b18-f0274234470d.png)
This error appeared when running in station mode. but the AP mode works fine with no issue. This error occurs in STA mode only. What can i do?

**Comment by StevenMHernandez** [comment]
What version of esp-idf are you using?

**Comment by Tremito96** [comment]
> 
> 
> What version of esp-idf are you using?

**Comment by Tremito96** [comment]
> 
> 
> What version of esp-idf are you using?

V 4.2

**Comment by Tremito96** [comment]
> 
> 
> What version of esp-idf are you using?

**Comment by StevenMHernandez** [comment]
Yeah, in its current state, the codebase is designed for v3.3.1 (It should be mentioned in the README: https://github.com/StevenMHernandez/ESP32-CSI-Tool#installation)

Can you use that version?

(Note to self: I need to update this project to use more recent esp-idf versions.)

**Comment by Tremito96** [comment]
V3.3.5 worked fine. Thank you very much. And i tried connecting to my smartphone's hot spot and also my home router. it says it connected but did not output any CSI info. Does this mean that my phone and router does not send csi information? (Welll i didnt have a second ESP32 to test it with two ESP32s)

**Comment by StevenMHernandez** [comment]
Try uncommenting this: https://github.com/StevenMHernandez/ESP32-CSI-Tool/blob/master/active_sta/main/main.c#L146

**Comment by Tremito96** [comment]
> 
> 
> Try uncommenting this: https://github.com/StevenMHernandez/ESP32-CSI-Tool/blob/master/active_sta/main/main.c#L146

yes. that was the problem. thank you very much
